public/src/main/java/com/revenuecat/purchases/models/StoreTransaction.kt

```diff
@@ -23,8 +23,17 @@ data class StoreTransaction(
     /**
      * Product Ids.
      */
+    @Deprecated(
+        "Replaced with productIds",
+        ReplaceWith("productIds")
+    )
     val skus: List<String>,
```

Comment:
As I mentioned in another comment, I think we can move this one away from the constructor and keep the `@Deprecated`. What do you think?

Reply:
yeah agreed!